src/app/containers/StoryPromo/index.jsx

```diff
@@ -68,13 +70,33 @@ StoryPromoImage.defaultProps = {
   }),
 };
 
+const LiveComponent = ({ headline, service, dir }) => (
+  // eslint-disable-next-line jsx-a11y/aria-role
+  <span role="text">
+    <LiveLabel service={service} dir={dir}>
```

Comment:
Does this have an aria-hidden attribute @OlgaLyubin

Reply:
And also just checking if this has had a developer accessibility swarm carried out on it?

Reply:
Yes this has an aria-hidden attribute - see https://github.com/bbc/psammead/blob/038a0bf5a4f2efd9792eaec0becec79cfbd56ad7/packages/components/psammead-story-promo/src/index.jsx#L260
Yes this has had a swarm - see https://github.com/bbc/psammead/pull/1359#issuecomment-516376021

Reply:
Fabulous, thank you @aacn500